Game_Manual/Levels_and_Roles.md

```diff
@@ -0,0 +1,156 @@
+
+Roles delineate different ways of interacting with the game. Players at all levels have the "Player" role, and as players advance they gain access to new roles.
+
+## Levels
+
+Levels are cumulative, so more advanced levels also include privileges of previous levels.
+
+Every level is a voting pool. In other words, players vote on goals that are in their level with other players in their level.
+
+Advancement is not one-way: players can move backwards if they stay in the red for two weeks (same mechanic as current in-the-red piece).
+
+### Level Abilities
+
+- Level 0
+  - Can access guild facilities
+  - Can choose `foundational` and `practice` goals
+  - Can use guild materials (e.g. borrow a laptop)
+- Level 1
+  - Can receive stipend
+  - Can create `foundational` and `practice` goals
+  - Can do cog app sessions with Technical Coaches
+- Level 2
+  - Can choose `apprentice` goals
+  - Can do cog app sessions with Project Leads
+- Level 3
+  - Can choose `prototype` goals
+  - Can create `apprentice` goals
+  - Can take on Technical Coach role
+  - Can earn Coaching stats (TBD)
+- Level 4
+  - Can choose `production` goals
+  - Can create `prototype` goals
+  - Can do mock technical interviews (TBD)
+- Level 5
+  - Can create `production` goals
+  - Can take on Project Lead role
+  - Can earn Leadership stats (TBD)
+
+#### A note on goal types
+
+There are 5 types of goals (only 3 of which are currently active). Eventually, there will be projects for each of the 5 types. Here's what each type means:
+
+1. `foundational`: no "product" produced. Purely for focusing on fundamental skill building. Example: "Relational DB design"
+1. `practice`: one-off, toy projects. For exploring and learning in a sandboxed environment, but with a defined (albeit contrived) "product". Example: "Bookstore App"
+1. `apprentice`: a real-world, maintained project, but without any user base or production environment. No need to manage a deployed app. Example: "Trossello"
+1. `prototype`: maintained projects with at least an "alpha-level" production environment and some user base. Will involve some dev ops. Example (hypothetical): a better goal library
+1. `production`: professional-grade projects with a live product and active users. Example (hypothetical): the LOS
+
+### Level Requirements
+
+This table shows each level starting at 0 and advancing to 5 with the criteria for advancing to that level.
+
+| Level | XP | Elo | Cultural | Team Play | Technical | Bias | Reviews | Other |
+|:-----------------------------------------------------------------------------|
+| 0 | - | - | - | - | - | - | - | enrolled in LG                               |
+| 1 | Δ/week>=17.5  | - | >=65 | >=65 | - | -8<=bias<=8 | - | joined a cohort  |
+| 2 | >=150 | >=1000 | >=80 | >=80 | - | - | >=20 | -                          |
+| 3 | >=500 | >=1040 | >=85 | >=85 | - | - | >=60 | -                          |
```

Comment:
I'm curious why we're letting players advance all the way to level 3, at which point they can be a technical coach, with no minimum technical health requirement. Is it really ok for someone who is consistently getting 0 for technical health to be a technical coach? Seems like in practice we'll want some cutoff.